moto/emr/responses.py

```diff
@@ -329,6 +331,8 @@ def run_job_flow(self):
                 ig["instance_count"] = int(ig["instance_count"])
                 # Adding support to EbsConfiguration
                 self._parse_ebs_configuration(ig)
+                # Adding support for auto_scaling_policy
```

Comment:
A generic "unflattening" of flattened querystring params, a la the immediately preceding EbsConfig special case, but using a generic mechanism.